### PR TITLE
HIVE-26326: Support enabling background threads when failover is in progress

### PR DESCRIPTION
**Pull request by hmangla98**

### CODE REVIEW

File: standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java [hunk 1]
```diff
@@ -291,9 +291,17 @@ public static boolean isTargetOfReplication(Database db) {
     return dbParameters != null && !StringUtils.isEmpty(dbParameters.get(ReplConst.TARGET_OF_REPLICATION));
   }
 
+  public static boolean forceEnableBackgroundThreads(Database db) {
```
Comment (pvary) [inline standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java:294]: is this `isBackgroundTreadsEnabled`?
The current name suggests for me that we set something forcefully, but the code shows that we are just checking a property

**Comment by pvary** [comment]
Is the test failure related?
If not, could you please check with the http://ci.hive.apache.org/job/hive-flaky-check/
So we can disable it if it is really flaky?

**Review by pvary — APPROVED** [review]
+1 pending tests

File: standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java [hunk 2]
```diff
@@ -291,9 +291,17 @@ public static boolean isTargetOfReplication(Database db) {
     return dbParameters != null && !StringUtils.isEmpty(dbParameters.get(ReplConst.TARGET_OF_REPLICATION));
   }
 
+  public static boolean isBackgroundThreadsEnabledForRepl(Database db) {
+    assert (db != null);
```
Comment (deniskuzZ) [inline standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java:295]: won't be executed in production code, should we throw an exception here?

**Pull request merged**